src/style.css

```diff
@@ -1,3 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+
+  --activation-color: #00ACDC;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding:0;
+}
+
+.nav__item {
+  margin: 0;
+  padding: 0 10px;
```

Comment:
remove paddings here to pass a tests, use one-directional margin for all items
margin-left: 20px and then remove it for :first-child